src/intersections/lines.jl

```diff
@@ -50,3 +54,21 @@ function intersectpoint(l1::Line{2}, l2::Line{2})
 
   a - s*v1
 end
+
```

Comment:
`intersectpoint` for 3D was missing, it's the same code as for 2D just taking into account the first two coordinates.
since the **purpose of intersectpoint** is not to check whether 2 objects intersect but to return the crossing point

Applying intersectpoint to objects that do not intersect would yield a Point which lies on the first line

Reply:
Can you please clarify the exact differences between the 2D and 3D implementations? By reading them quickly with the naked eyes I am seeing no differences. If they are the same implementation, maybe we just need to relax the type of the arguments instead of duplicating code?